frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -18,7 +18,13 @@
                 <b-icon icon="three-dots-vertical"></b-icon>
               </template>
 
-              <b-dropdown-item v-if="validLink" class="test-copy-link" @click="copy">
+              <b-dropdown-item
+                v-if="validLink"
+                class="test-copy-link"
+                v-clipboard="link"
+                @success="toastSuccess($t('gdd_per_link.link-copied'))"
+                @error="copyLink"
```

Comment:
are you sure you fix the problem this way?

Reply:
no

Reply:
LOL